web/modules/custom/og_comment/src/OgCommentFieldItemList.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\og_comment;
+
+use Drupal\comment\CommentFieldItemList;
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Session\AccountInterface;
+
+/**
+ * Defines a item list class for comment fields.
+ */
+class OgCommentFieldItemList extends CommentFieldItemList {
+
+  /**
+   * The og access manager service.
+   *
+   * @var \Drupal\og\OgAccessInterface
+   */
+  protected $ogAccess;
+
+  /**
+   * The config factory service.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function access($operation = 'view', AccountInterface $account = NULL, $return_as_object = FALSE) {
+    $host_entity = $this->getEntity();
+    // We cannot inject services in this plugin as it extends TypedData and it
+    // does not support dependency injection.
+    // @see https://www.drupal.org/node/2053415
+    $this->configFactory = \Drupal::service('config.factory');
+    $this->ogAccess = \Drupal::service('og.access');
+    $account = $account ?: \Drupal::currentUser();
+    $comment_access_strict = $this->configFactory->get('og_comment.settings')->get('comment_access_strict');
+
+    if ($operation === 'edit') {
+      // Only users with administer comments permission can edit the comment
+      // status field.
+      $result = $this->hasPermission('administer comments', $host_entity, $account);
+      return $return_as_object ? $result : $result->isAllowed();
+    }
+    if ($operation === 'view') {
+      // Only users with either post comments or access comments permisison can
```

Comment:
Oh! This typo is inherited from core! This could be a really quick contribution :)